quickscreen/analysis/datafill.py

```diff
@@ -56,8 +59,12 @@ def columntype(self, column):
         numpy.dtype
             dtype of column
         """
-        assert (isinstance(column, int) or column in self.data), "Please enter an integer or the name of a column"
-        return self.data.dtypes[column]
+        try:
+            assert (isinstance(column, int) or column in self.data)
+            return self.data.dtypes[column]
+        except:
```

Comment:
Its generally frowned upon to not catch specific errors.
Although this try and except block is simple, having try excepts without specific errors can hide trivial bugs easily. 

Also, [lab4.md](https://github.com/ubco-mds-2020/data_533/blob/master/labs/lab4/lab4.md) say `At least six methods will contain appropriate error and exception handlers` which in my reading is catching specific errors.

Reply:
adding to Mitch's comments, you can just change your code to be `except AssertError`, or you can create your own user defined error class and do something along the lines of `if not is instance(x, y) raise UserErrorClass()` then you can use `except UserErrorClass:`